examples/quick_start/scripts/inference/README.md

```diff
@@ -0,0 +1,73 @@
+# Deploying a Ranking model on Triton Inference Server
+The last step of ML pipeline is to deploy the trained model into production. For this purpose we use NVIDIA Triton Inference Server](https://github.com/triton-inference-server/server), which is an open-source inference serving software, standardizes AI model deployment and execution and delivers fast and scalable AI in production. 
+
+[Merlin Systems](https://github.com/NVIDIA-Merlin/systems/tree/main) library is designed for building pipelines to generate recommendations. Deploying pipelines on Triton is one part of the library's functionality and Merlin Systems provides easy to use APIs to be able to export ensemble graph and model artifacts so that they can be loaded on Triton with less effort.
+
+In this example we demonstrate the necessary steps to deploy a model to Triton and test it:
+
+1. Creating the ensemble graph
+2. Launching the Triton Inference Server
+3. Sending request to Server and receiving the response
+
+## Creating the Ensemble Graph
+
+In order to do model deployment stage, you are required to complete `preprocessing` and `ranking` steps already.  At the inference step, we do have a collection of multiple (individual) models to be deployed on Triton. We deploy NVTabular model as well to be able to transform raw data as we do in the preprocessing phase. In this context, deploying multiple models is called an ensemble model since it represents a pipeline of one or more models and the connection of input and output tensors between those models. Ensemble models are intended to be used to encapsulate a procedure that involves multiple models, such as "data preprocessing -> inference -> data postprocessing". 
+
+The Triton Inference Server serves models from one or more model repositories that are specified when the server is started. Each model in a model repository must include a model configuration that provides required and optional information about the model. Merlin Systems simplified that step, so that we can easily export ensemble graph config files and artifacts. We use [Ensemble](https://github.com/NVIDIA-Merlin/systems/blob/main/merlin/systems/dag/ensemble.py#L29) class for that. The class is responsible for interpreting the graph and exporting the correct files for the Triton server.
```

Comment:
```suggestion
The Triton Inference Server serves models from one or more model repositories that are specified when the server is started. Each model must include a configuration that provides required and optional information about the model. Merlin Systems simplified that step, so that we can easily export ensemble graph config files and artifacts. We use [Ensemble](https://github.com/NVIDIA-Merlin/systems/blob/main/merlin/systems/dag/ensemble.py#L29) class for that, which is responsible for interpreting the graph and exporting the correct files for the Triton server.
```